packages/block-library/src/post-author/block.json

```diff
@@ -35,6 +35,7 @@
 	},
 	"usesContext": [ "postType", "postId", "queryId" ],
 	"supports": {
+		"inserter": true,
```

Comment:
When this PR is ready to be merged, I'm going to change this value to `false`.

Reply:
If you'd like to get it updated sooner, I don't think it complicates testing too much to add a sample snippet of block markup that could be pasted into the code editor, selected and configured further.

```
<!-- wp:post-author {"avatarSize":96,"showBio":true,"byline":"Custom byline.","isLink":true,"linkTarget":"_blank"} /-->
```

Reply:
Indeed. Updated with 7390397. I will also update testing instructions and screencasts 👍